sysutils/libfaketime/Portfile

```diff
@@ -4,12 +4,11 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           muniversal 1.1
 PortGroup           legacysupport 1.1
+PortGroup           makefile 1.0
 
 github.setup        wolfcw libfaketime 0.9.10 v
-revision            1
-
+revision            2
```

Comment:
this doesn't change anything consequential do there is no need to increase the "revision". If anything, I would make changes like this when updating the version...

Reply:
I'd forgotten about removing the patch that the port previously used. Does that count for a rev-bump since the build would be different?

Reply:
Presumably the implementation in legacy-support would do the same thing? I didn't check, but let's see what @catap thinks about this change.

Reply:
@reneeotten yes, the new version of legacy support includes the fix, seems that no need for a patch.

@TheRealKeto have you tried it on some old system? Does it work?

Anyway, I'd like to keep revbump to force rebuild and relink against new legacy-support.

Reply:
@catap Yes. I tried building the project on OSX Lion 10.7.5. More information is on the description of the PR. I can't vouch for other legacy systems, but given that it works on as low as Lion, it's good to go :)

Reply:
@TheRealKeto just build or also use it? :)

Reply:
@catap I tested the `faketime` binary after building and installing with a few dates (e.g `faketime -f "1999-01-01 00:00:00"`), as well as whether it could still "hook" into system binaries (given that I was also on OSX Lion)